src/containers/LoginFormContainer.js

```diff
@@ -38,47 +43,50 @@ export default class LoginFormContainer extends Component {
 
     this.setState({
       isLoggedIn: true,
-      userID: response.profileObj.googleId,
+      token: response.accessToken,
       firstName: response.profileObj.givenName,
       lastName: response.profileObj.familyName,
       email: response.profileObj.email,
       birthday: response.birthday,
-      picture: response.profileObj.imageUrl
+      picture: response.profileObj.imageUrl,
+      externalProviderName: 'Google',
+      roleName: 'Normal'
+
     });
+    this.addNewUser();
   };
 
   addNewUser = () => {
-    axios
-      .post(`${BASE_URL}/login`, {
-        firstName: this.state.first_name,
-        lastName: this.state.last_name,
+    axios.post(`${BASE_URL}/login`, {
+        firstName: this.state.firstName,
```

Comment:
you can use destructuring state, code will look better:
**const { picture, firstName} = this.state;** 
and below you can use just picture without repeating this.state
please do it with all properties used here